src-tauri/src/main.rs

```diff
@@ -51,18 +55,20 @@ async fn get_macros(state: tauri::State<'_, MacroBackend>) -> Result<MacroData,
 async fn set_macros(
     state: tauri::State<'_, MacroBackend>,
     frontend_data: MacroData,
-) -> Result<(), ()> {
-    state.set_macros(frontend_data).await;
-    Ok(())
+) -> Result<(), String> {
+    state
+        .set_macros(frontend_data)
+        .await
+        .map_err(|err| err.to_string())
 }
 
 #[tauri::command]
-async fn get_monitor_data() -> Result<Vec<plugin::system_event::Monitor>, ()> {
-    #[cfg(any(target_os = "windows", target_os = "linux"))]
-    return Ok(plugin::system_event::backend_load_monitors().await);
-
-    #[cfg(target_os = "macos")]
-    return Ok(vec![]);
+async fn is_debug() -> bool {
+    if let Ok(result) = std::env::var(DEBUG_ENVVAR) {
+        return log::LevelFilter::from_str(result.as_str()).unwrap() >= log::LevelFilter::Debug;
```

Comment:
Ideally, in this case it would try and parse it, if it doesn't, then it should give a warning log indicating that the env variable is defined, but defined incorrectly

Reply:
Yeah the idea was that you shouldnt boot up the app with incorrect log level because you already make an effort to change the log level and then not noticing it doesn't work because it's not defined properly would be bad.

Ideally i'd show a frontend toast but not sure if thats not way out of scope. I can change it to do an error log write instead

Reply:
Yeah, just do an error log. I don't think this code is really going to help, because if they have the env variable bad, they'll just get a hard crash, likely with no context, so how will they know that that is the issue?

Reply:
Changed it a bit, now has a notification on frontend toast instead: [8465165](https://github.com/WootingKb/wooting-macros/pull/194/commits/8465165b557c25840934d33d8d35f8ca6254704d)

Reply:
The notification: 
![image](https://github.com/WootingKb/wooting-macros/assets/1900179/18faf486-1ff8-43a0-b6e2-4592ce7f8574)

Reply:
![image](https://github.com/WootingKb/wooting-macros/assets/1900179/75e47dd8-ab55-40ad-a3fe-865a9893a041)


Tweaked: [df1952f](https://github.com/WootingKb/wooting-macros/pull/194/commits/df1952fe4dca0101b77b559bc6afb077ca88b842)